tr_tr/_messages.js

```diff
@@ -0,0 +1,1397 @@
+// -*- mode: java; c-basic-offset: 2; -*-
+// Copyright © 2017 Pavitra, Makeroid. All rights reserved.
+
+/**
+ * Visual Blocks Language
+ *
+ * Copyright © 2017 Pavitra, Makeroid. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Traditional English strings.
+ * @author mckinney@gmail.com (Andrew F. McKinney)
+ */
+'use strict';
+
+goog.provide('AI.Blockly.Msg.en');
```

Comment:
You have to set `goog.provide('AI.Blockly.Msg.tr');`

Reply:
done